crates/sui-indexer/migrations_v2/2023-08-19-044020_events/up.sql

```diff
@@ -0,0 +1,24 @@
+CREATE TABLE events
+(
+    tx_sequence_number          BIGINT       NOT NULL,
+    event_sequence_number       BIGINT       NOT NULL,
+    transaction_digest          bytea        NOT NULL,
+    checkpoint_sequence_number  bigint       NOT NULL,
+    -- array of SuiAddress in bytes. All signers of the transaction.
+    senders                     bytea[]      NOT NULL,
```

Comment:
Note for us on GraphQL that we need to reflect the fact that transactions can have multiple senders in `Event` and `TransactionBlock` in the GraphQL schema.

Reply:
that's why it's `bytea[]` or am i misunderstanding what you meant?

Reply:
This is a comment for the folks working on GraphQL that we need to update our schema to make use of the fact that this is an array (Currently GraphQL's `TransactionBlock` only has a scalar `sender: SuiAddress` field, not `senders: SuiAddress`).